packages/new-polymath-ui/src/components/Table/docs/makeData.ts

```diff
@@ -25,8 +25,8 @@ const newPerson = () => {
   };
 };
 
-export function makeData(...lens) {
-  const makeDataLevel = (depth = 0) => {
+export function makeData(...lens: any[]) {
```

Comment:
Shouldn't `lens` be typed as `number[]`?

Reply:
Wont fix on this PR this is demo code